runtimes/rust/src/api.rs

```diff
@@ -14,7 +14,7 @@
 
 //! Building blocks for creating HTTP API of Rust services.
 
-pub use exonum_api::{Deprecated, EndpointMutability, Error, FutureResult, Result};
+pub use exonum_api::{Deprecated, EndpointMutability, Error, FutureResult};
```

Comment:
Why is `Result` no longer re-exported? As far as I see, this is the only thing forcing `cryptocurrency-advanced` and some other crates to have `exonum-api` as the direct dependency.